src/components/Verify.tsx

```diff
@@ -90,16 +82,26 @@ export const Verify = () => {
                 htmlFor='repoCommit'
                 className='block text-sm font-medium leading-6 text-gray-900'
               >
-                Commit Hash
+                Commit Hash<span className='text-red-600'> *</span>
               </label>
             </div>
             <div>
               <input
                 id='repoCommit'
-                onBlur={handleInput}
-                required
-                className='block w-full rounded-md border-0 py-1.5 pl-2 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6'
+                className='block w-full rounded-md border-0 py-1.5 pl-2 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-900 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6'
+                {...register('repoCommit', {
+                  required: 'This is required',
```

Comment:
Can we add a validation that's a hex string (no 0x prefix)

Reply:
Add hex validation https://github.com/ScopeLift/cove-frontend/pull/9/commits/7ed9487c39b584dbde2db6808ce3c1e1da3d0600
Add 0x check https://github.com/ScopeLift/cove-frontend/pull/9/commits/655620fbf7d9fbd94340548104cdd50893b1725d